scripts/setup_synapse_organization.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python3
+"""Setup HTAN2 JSON Schema Organization in Synapse.
+
+This script creates the HTAN2 organization for JSON schemas and sets up
+the initial folder structure for data validation.
+"""
+
+import argparse
+import synapseclient
+import os
+from pathlib import Path
+
+
+def setup_synapse_organization(org_name: str = "HTAN2", create_folders: bool = True):
+    """Set up the HTAN2 organization and folder structure in Synapse.
+    
+    Args:
+        org_name: Name of the organization to create
+        create_folders: Whether to create initial folder structure
+    """
+    
+    # Configure Synapse client for production stack
+    syn = synapseclient.Synapse()
+    syn.setEndpoints(**synapseclient.client.PRODUCTION_ENDPOINTS)
+    
+    print(f"Configuring for production stack")
+    
+    # Login to Synapse
+    print("Logging in to Synapse...")
+    syn.login()
+    print("Successfully logged in to Synapse")
+    
+    # Get available services
+    syn.get_available_services()
+    schema_service = syn.service("json_schema")
+    
+    # Create or get the organization
+    print(f"Setting up organization: {org_name}")
+    try:
+        schema_org = schema_service.JsonSchemaOrganization(name=org_name)
+        schema_org.create()
```

Comment:
Nit: the setting up of the HTAN2 jsonschema organization is a precursor to the project setup, but "independent". Since you create the jsonschema organization once, consider having the "project bootstrap" code as separate functions.

So if we wanted to control the entire project infrastructure as code, you can consider something like

```
center: ...
top_level_folder_name:
       - component...
 ```

But this is a bit risky because there's no "drift protection" within Synapse, so anyone can edit the folder names in Synapse which would make this out of sync.  So then the idea would be to bootstrap projects, THEN as we discussed, have a yaml config that stores synapse id <-> jsonschema URI mappings to do the binding